dedupe/predicates.py

```diff
@@ -71,7 +67,7 @@ def __len__(self) -> int:
         return 1
 
     @abc.abstractmethod
-    def __call__(self, record: RecordDict, **kwargs) -> Iterable[str]:
+    def __call__(self, record: RecordDict, **kwargs) -> FrozenSet:
```

Comment:
the should be Frozenset[str] here and everywhere else

Reply:
Done